examples/analysis/default.asm

```diff
@@ -0,0 +1,8 @@
+// For llvm-mca a -mcpu=... argument is required
+// Try -mcpu=haswell or -mcpu=btver2
```

Comment:
The problem with doing anything super specific there is that it would be harder to use for other tools in general (which probably wouldn't care about -mcpu= (if they accepted it at all)).

What do you think?

Reply:
Derp, of course. I'll write an llvm-mca subtotal which changes `filterUserOptions` (and adds `-mcpu=btver2` if none was passed) unless someone suggests a better one.

Thank you,
 Filipe